app/controllers/application_controller.rb

```diff
@@ -9,6 +9,10 @@ class ApplicationController < ActionController::Base
 
   rescue_from ActionView::MissingTemplate, with: :routing_error
 
+  rescue_from RateLimitChecker::LimitReached do |exc|
```

Comment:
is `exc` the actual error? (curious why we didn't go with `e` or `err` or `error` here...

asking because `exc` makes me think it's not an error object

Reply:
I believe that's `exception` ?

Reply:
Yes, exc is the error or exception. This is how we had it in the API controller and I literally just moved the code over here without messing with it 🤷@rhymes is the fan of exec, I'm good with whatever.

Reply:
Ok, good to know. Don't bother with it, I was just curious.